pino.js

```diff
@@ -115,12 +116,13 @@ Object.defineProperty(pino, 'levels', {
   enumerable: true
 })
 
-function Pino (level, stream, serializers, stringify, end, name, slowtime, chindings, cache, formatOpts) {
+function Pino (level, stream, serializers, stringify, end, name, timestamp, slowtime, chindings, cache, formatOpts) {
```

Comment:
They are different features. How do you propose reducing the verbosity? I would use an object for a single parameter to this function, but I imagine it's done this way for speed.